docs/03-github/03-test-runner.mdx

```diff
@@ -329,7 +329,6 @@ jobs:
           artifactsPath: ${{ matrix.testMode }}-artifacts
           githubToken: ${{ secrets.GITHUB_TOKEN }}
           checkName: ${{ matrix.testMode }} Test Results
-          coverageOptions: 'generateAdditionalMetrics;generateHtmlReport;generateBadgeReport'
```

Comment:
hmm actually, I think it might be good to also show exact syntax to support this so I'm not sure it should be completely removed.

Reply:
Sure, the example they provide in the docs is this argument:

```
"generateAdditionalMetrics;generateHtmlReport;generateHtmlReportHistory;generateBadgeReport;
assemblyFilters:+my.assembly.*,+<packages>;pathFilters:-**/Tests/**,-**/BuiltInPackages/**;verbosity:verbose"
```

So maybe cutting this down to just `assemblyFilters:+my.assembly.*` should be enough :D